src/ActionManager.js

```diff
@@ -175,21 +176,21 @@ export default class ActionManager {
    * @returns {Array}
    */
   static filterMessages(messages, trigger='', verb=null, noun='', params={}) {
```

Comment:
Don't forget to modify JSDocs for this function

Reply:
I think you can keep the if and just change the name to `triggers` since the var internally represents an array of triggers.